src/corehost/cli/apphost/CMakeLists.txt

```diff
@@ -20,8 +20,13 @@ set(SKIP_VERSIONING 1)
 
 set(SOURCES
     ../fxr/fx_ver.cpp
+    ./bundle/bundle_runner.cpp
+    ./bundle/manifest.cpp
+    ./bundle/file_entry.cpp
 )
 
+include_directories(./bundle)
+
 set(HEADERS
     ../fxr/fx_ver.h
```

Comment:
Please add the header files as well. It's not needed for the build itself, but it helps make the generated VS solution much easier to use.